tutorials/Coordinates/Coordinates.ipynb

```diff
@@ -0,0 +1,781 @@
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "If the object you're interested in is in SIMBAD, you can also look it up directly from its name using the `SkyCoord.from_name()` [class method](https://julien.danjou.info/blog/2013/guide-python-static-class-abstract-methods). \n",
+      "\n",
+      "(Note that this requires an internet connection.  It's safe to skip if you don't have one, because we defined it above explicitly.)"
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "hcg7_center = SkyCoord.from_name('HCG 7')\n",
+      "hcg7_center"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "This object we just created has various useful ways of accessing the information contained within it.  In particular, the ``ra`` and ``dec`` attributes are specialized ``Quantity`` objects that store angles and provide pretty representations of those angles and some useful attributes to quickly convert to common angle units:"
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "type(hcg7_center.ra), type(hcg7_center.dec)"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "hcg7_center.dec"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "hcg7_center.ra"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "hcg7_center.ra.hour"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "Now that we have a `SkyCoord` object, we can start using it to access data from the [Sloan Digitial Sky Survey](http://www.sdss.org/).  Lets start by just trying to get a picture using the SDSS image cutout service."
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "impix = 1024\n",
+      "imsize = 12*u.arcmin\n",
+      "cutoutbaseurl = 'http://skyservice.pha.jhu.edu/DR12/ImgCutout/getjpeg.aspx'\n",
+      "cutouturl = cutoutbaseurl + '?' + urllib.urlencode(dict(ra=hcg7_center.ra.deg, dec=hcg7_center.dec.deg, \n",
+      "                                                        width=impix, height=impix, \n",
+      "                                                        scale=imsize.to(u.arcsec).value/impix))\n",
+      "urllib.urlretrieve(cutouturl, 'HCG7_SDSS_cutout.jpg')"
```

Comment:
Hmm, yeah, can you attach a screenshot?  It looks pretty clean in my browser so it would be good to know what comes out hard-to-see for you.

As for `download_file`, it won't work in its current form.  The intent here is to actually save the file *locally* so that the next cell works (which *also* is supposed to work with the pre-supplied image in the same directory).  Currently `download_file` only saves to the astropy cache or a local temporary file, and can't save to a particular name.   

As to whether it should *gain* that functionality, or query parameters: I'm not sure it should.  This problem is already solved by the standard library (or `requests` is what I would normally use), so no need for astropy to re-invent the wheel.  The one missing piece is the progress bar, though... *that* might be a reason to include it (and we could easily enough update this tutorial if that change is made).